app/assets/stylesheets/_variables.scss

```diff
@@ -9,3 +9,5 @@ $autolab-black-text: #212121;
 $autolab-blue-text: #0869af;
 $autolab-green: #3a862d;
 $autolab-white: #fff;
+$autolab-subtle-gray: #6f6f6f;
+$autolab-highlight-gray: #0869af;
```

Comment:
nit: newline

Reply:
Can you also rename these variables? `autolab-subtle-gray` and `autolab-highlight-gray` already exist in `_variables.scss` so this is breaking existing styles

![Screenshot 2024-01-27 at 1 28 53 PM](https://github.com/autolab/Autolab/assets/25730111/5638c54a-56fa-4036-91db-045aa24e2646)